data/org.freedesktop.impl.portal.Settings.xml

```diff
@@ -43,6 +43,23 @@
           Unknown values should be treated as 0 (no preference).
         </para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term>org.freedesktop.appearance window-decorations s</term>
+        <listitem><para>
+          Arrangement of buttons on the titlebar.
+          Supported values are:
+          <simplelist>
+            <member>menu: application content, or titlebar</member>
+            <member>minimize: the minimize button</member>
+            <member>maximize: the maximize button</member>
+            <member>close: the close button</member>
+          </simplelist>
+          Unknown values should be ignored. If a value is duplicated, it should be drawn again in the indicated area.
```

Comment:
I think it would make sense to have this before the list of supported buttons. Right now, it looks kind of like it's describing an enum, rather than a string containing the button layout.

Reply:
Wouldn't it also make sense to change the placement of the description on color-scheme too, then? Trying to understand how you'd ideally want it to be laid out.

Reply:
Well, color-scheme is describing an enum value so it's a bit different imo :p